packages/flutter/test/cupertino/checkbox_test.dart

```diff
@@ -883,3 +958,23 @@ void main() {
     await tester.pump();
   });
 }
+
+class CheckboxMouseCursor extends WidgetStateMouseCursor {
```

Comment:
Nit: Make this private? When I saw it used in the test above, I wondered if it was something in the actual framework.